src/Listener/ViewListener.php

```diff
@@ -318,18 +318,23 @@ protected function _getControllerActions()
 
         $actions = $this->_getAllowedActions();
         foreach ($actions as $actionName) {
-            $action = $this->_action($actionName);
             $method = 'GET';
-            $class = get_class($action);
-            $class = substr($class, strrpos($class, '\\') + 1);
-            $scope = $action->scope();
-
-            if ($class === 'DeleteAction') {
-                $method = 'DELETE';
-            }
+            try {
+                $action = $this->_action($actionName);
+                $class = get_class($action);
+                $class = substr($class, strrpos($class, '\\') + 1);
+                $scope = $action->scope();
+
+                if ($class === 'DeleteAction') {
+                    $method = 'DELETE';
+                }
 
-            if ($class === 'AddAction') {
-                $scope = 'table';
+                if ($class === 'AddAction') {
+                    $scope = 'table';
+                }
+            } catch (\Crud\Error\Exception\ActionNotConfiguredException $e) {
+                // fallback: assume $actionName is a standard cake's controller action
+                $scope = 'entity';
```

Comment:
being able to provide the `$scope` seem nice - maybe if `$actionName` is array, expect two keys `action` and `scope` ?